src/components/sst-key.tsx

```diff
@@ -47,17 +48,14 @@ const renderTemperatureLabels = (increments: number, tempScaleName: string) => {
 @inject("stores")
 @observer
 export class SSTKey extends BaseComponent<IBaseProps, {}> {
-  public toggleColorBlindKey = () => {
-    const newValue = !this.stores.ui.colorBlindSSTScale;
-    this.stores.ui.setColorBlindSSTScale(newValue);
+  public preventClickPropagation = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
Definitely. :) The native <input> was using separate onClick and onChange as otherwise React was complaining about onChange not being used (readOnly input). I accidentally left the wrong name, thanks. Just fixed in the last commit.